.github/workflows/functional-test.yaml

```diff
@@ -414,7 +413,7 @@ jobs:
           # AZURE_OIDC_ISSUER
           eval "export $(echo "${{ secrets.FUNCTEST_AZURE_OIDC_JSON }}" | jq -r 'to_entries | map("\(.key)=\(.value)") | @sh')"
 
-          AUTHKEY=$(echo -n "${{ github.actor }}:${{ secrets.GH_RAD_CI_BOT_PAT }}" | base64)
+          AUTHKEY=$(echo -n "${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}" | base64)
```

Comment:
Does this work? What's this token being used for? I would expect the github token to no longer be valid once the test run is over.

Reply:
I can remove this - only had it there for fork testing.